package.json

```diff
@@ -31,34 +33,49 @@
     "url": "https://github.com/react-navigation/react-navigation-hooks/issues"
   },
   "homepage": "https://github.com/react-navigation/react-navigation-hooks#readme",
+  "@react-native-community/bob": {
+    "source": "src",
+    "output": "dist",
+    "targets": [
+      [
+        "commonjs",
+        {
+          "flow": true
```

Comment:
This won't do anything if you don't use Flow

Reply:
Thanks will remove it :)